### ISSUE TITLE
Concurrent `PurgeOrchestrationInstanceHistoryAsync` calls

### ISSUE THREAD

**Issue by FarhadJabiyev** [microsoft/durabletask-netherite#418]
We should call `PurgeOrchestrationInstanceHistoryAsync` method to delete orchestrations started more than 24 hours ago.  We want to make that call from `BackgroundService` in .NET core. In documentation nothing mentioned if we can do that call from multiple services containing that `BackgroundService`  or not. Could you please let me  know if it safe to make that call concurrently with same arguments or not?

**Comment by sebastianburckhardt** [comment]
Yes, the call is safe to make concurrently and/or repeatedly with the same arguments.
